services/workflows-service/src/user/user.controller.internal.ts

```diff
@@ -67,5 +69,29 @@ export class UserControllerInternal {
       },
       projectIds?.[0] || currentProjectId,
     );
+    if (shouldSendWelcomeEmail && createdUser.email) {
```

Comment:
A space would help readability here. 🙂 
```suggestion
    );
    
    if (shouldSendWelcomeEmail && createdUser.email) {
```